File: Python/Entities/DeleteEntityObject.py
import requests

boards_on_fire_api_url = "YOUR URL"  # example: https://my-company.boardsonfireapp.com/api/v5/
api_key = "YOUR API KEY"
entity_name = "YOUR ENTITY NAME"
entity_object_id = "YOUR ENTITY OBJECT ID"

headers = {
    "x-api-key": api_key
}

try:
    response = requests.delete(f"{boards_on_fire_api_url}/entities/{entity_name}/entityObjects/{entity_object_id}", headers=headers)
    response.raise_for_status()
    print("EntityObject deleted successfully.")
except requests.exceptions.RequestException as e:
    print(f"Request error: {e}")

File: Golang/Entities/DeleteEntityObject.go
package main

import (
    "fmt"
    "net/http"
)

func main() {
    boardsOnFireApiUrl := "YOUR URL" // example: https://my-company.boardsonfireapp.com/api/v5/
    apiKey := "YOUR API KEY"
    entityName := "YOUR ENTITY NAME"
    entityObjectId := "YOUR ENTITY OBJECT ID"

    client := &http.Client{}

    req, err := http.NewRequest("DELETE", fmt.Sprintf("%s/entities/%s/entityObjects/%s", boardsOnFireApiUrl, entityName, entityObjectId), nil)
    if err != nil {
        fmt.Println("Error creating request:", err)
        return
    }

    req.Header.Set("x-api-key", apiKey)

    resp, err := client.Do(req)
    if err != nil {
        fmt.Println("Request error:", err)
        return
    }
    defer resp.Body.Close()

    if resp.StatusCode != http.StatusOK {
        fmt.Println("Request failed with status:", resp.Status)
        return
    }

    fmt.Println("EntityObject deleted successfully.")
}
